Tweeter/Controllers/HomeController.cs

```diff
@@ -3,13 +3,33 @@
 using System.Linq;
 using System.Web;
 using System.Web.Mvc;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.AspNet.Identity;
+using Tweeter.Models;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Tweeter.DAL;
 
 namespace Tweeter.Controllers
 {
     public class HomeController : Controller
     {
+        protected ApplicationDbContext ApplicationDbContext { get; set; }
```

Comment:
mmm this all looks funky... you should only need to instantiate your dbContexts in the repos